pkg/apis/config/features.go

```diff
@@ -66,6 +66,7 @@ const (
 	FeaturePodSpecHostPID                   = "kubernetes.podspec-hostpid"
 	FeaturePodSpecHostPath                  = "kubernetes.podspec-volumes-hostpath"
 	FeaturePodSpecInitContainers            = "kubernetes.podspec-init-containers"
+	FeaturePodSpecMountPropagation          = "kubernetes.podspec-mount-propagation"
```

Comment:
Should not we add this to the configmap (features.yaml)?

Reply:
i'll add